doc/sphinx/source/frameworks/index.rst

```diff
@@ -82,10 +86,12 @@ updates.
 .. [#1] Benchmarking performance of DL systems is a young discipline; it is a
    good idea to be vigilant for results based on atypical distortions in the 
    configuration parameters. Every topology is different, and performance 
-   increases or slowdowns can be attributed to multiple means.    
+   increases or slowdowns can be attributed to multiple means.  Also watch out 
```

Comment:
`increases or slowdowns can be attributed to multiple means` -> `changes can be attributed to multiple causes`

Reply:
Fixed